.gitignore

```diff
@@ -7,5 +7,4 @@
 npm-debug.log
 
 *.js.map
-/src/**/*.js
```

Comment:
we have a little JS now, and we are no longer compiling garbage js into folders so this is no longer necessary